File: C++/Estrutura Sequencial/Terreno.cpp
#include <iostream>
#include <iomanip>
#include <string>
#include <climits>
#include <cmath>

using namespace std;

int main()
{

    double largura, comprimento, valor, preco, area;

    cout << "Digite a largura do terreno: ";
    cin >> largura;
    cout << "Digite o comprimento do terreno: ";
    cin >> comprimento;
    cout << "Digite o valor do metro quadrado: ";
    cin >> valor;

    area = largura * comprimento;
    preco = area * valor;

    cout << fixed << setprecision(2);
    cout << "Area do terreno = " << area << endl;
    cout << "Preco do terren = " << preco << endl;

    return 0;
}

File: Java/Estrutura Sequencial/Terreno.java
import java.util.Locale;
import java.util.Scanner;

public class Terreno {
    public static void main(String[] args) {

        Locale.setDefault(Locale.US);
        Scanner sc = new Scanner(System.in);

        double largura, comprimento, area, valor, preco;

        System.out.print("Digite a largura do terreno:");
        largura = sc.nextDouble();

        System.out.print("Digite o comprimento do terreno: ");
        comprimento = sc.nextDouble();

        System.out.print("Digite o valor do metro quadrado: ");
        valor = sc.nextDouble();

        area = largura * comprimento;

        preco = valor * area;

        System.out.println("Area do terreno = " + String.format("%.2f", area));
        System.out.println("Preco do terreno = " + String.format("%.2f", preco));

        sc.close();

    }
}
